README.md

```diff
@@ -84,11 +92,6 @@ very latest API version. As of this writing, the site is compatible with API ver
     mix deps.get
     ```
 
-1. Install npm globally
-   ```
-   npm install -g npm@6.7.0
```

Comment:
I think we need to explicitly all use this version. I recall lots of unintended package-lock changes due to people having the wrong or different versions in different commits. See https://github.com/mbta/dotcom-legacy/pull/2825

Reply:
Node _may_ have fixed this since then, but not 100% sure.

Reply:
I've just checked, and the version of `npm` (6.4.1) that ships with our current version of Node does not generate the lockfile any differently than `npm` 6.7.0, at least on my machine.

However they do _both_ generate the lockfile slightly differently than the current committed version; a few packages listed as `optional` become non-optional. The difference appears to be the same whether `npm` 6.4.1 or 6.7.0 is used. I'll PR this change in a separate branch if we think it's harmless (I think it likely is, from experience).

Reply:
Update: The separate PR was closed for now. Let's keep this as is (explicitly requiring this version of npm), and revisit a little later on if new devs continue to experience discrepancies.